tests/phpunit/elementor/includes/template-library/test-manager-local.php

```diff
@@ -141,4 +142,67 @@ public function test_should_return_data_from_get_template_data() {
 
 		$this->assertEquals( $ret, [ 'content' => [] ] );
 	}
+
+	public function test_get_data__document_without_data() {
+		$is_edit_mode = Plugin::$instance->editor->is_edit_mode();
+
+		Plugin::$instance->editor->set_edit_mode( false );
+
+		$post = $this->factory()->create_and_get_custom_post( [
+			'meta_input' => [
+				'_elementor_data' => [],
+			],
+		] );
+
+		$result = Plugin::$instance->templates_manager->get_source( 'local' )->get_data( [
+			'template_id' => $post->ID,
+		] );
+
+		$this->assertArrayHasKey( 'content', $result );
+		$this->assertEmpty( $result['content'] );
```

Comment:
Assers it's an array.

Reply:
`assertIsArray()` is not exists in php 5.6. so I used `assertIsTrue(is_array($result['content']))`